packages/legend-studio-plugin-query-builder/src/components/QueryBuilderFetchStructurePanel.tsx

```diff
@@ -78,27 +78,25 @@ export const QueryBuilderFetchStructurePanel = observer(
             <div className="panel__header__title__label">fetch structure</div>
           </div>
           <div className="panel__header__actions">
-            {queryBuilderState.TEMPORARY__enableGraphFetch && (
```

Comment:
One thing about removing this. are we losing any features by switching to the graph fetch in query builder? No right as both graph fetch trees should be the same. wouldnt want to lose key features like "select all mapped properties".